mobile/lib/src/domain/settings/acquisition_board.dart

```diff
@@ -0,0 +1,3 @@
+part of 'settings.dart';
+
+enum AcquisitionBoard { Empty, Cython, Ganglion }
```

Comment:
@WilliamHarvey97 @MouradLachhab y a-t-il moyen de détecter le device automatiquement?

Reply:
Exemple, y aurait-il un UID que les devices ont? On pourrait par exemple détecter que le device n'est pas supporté si c'est pas un Ganglion ou un Cyton.

Reply:
Pour ce qui est du cyton, on ne communique pas directement avec, on communique juste avec le usb_serial. En ce moment on filtre déjà tout ce qui n'est pas un FTDI chip, donc dans la liste des device available on voit seulement celui-ci. 

Pour ce qui du ganglion, on a juste un MAC adress et un nom. On reçoit pas les informations du vendeur ou manufacturer. Cependant so far pour notre ganglion, le nom est Ganglion-d664, donc il est possible que tous les ganglions contiennent le string ganglion dans leur nom, mais je pourrais pas confirmer.

La raison principale pourquoi on fait pas en ce moment la détection automatique c'est que le scan for device se fait soit sur le usb, soit sur le bluetooth et non les deux à la fois. D'après moi ça pourrait être changé si on veut le faire automatiquement.

Reply:
Avec la nouvelle information obtenue aujourd'hui, il n'est pas important de connaitre si le board est un Cyton ou Ganglion. Seul le sample rate est important.